### PR TITLE
Fix unique container name for splunk-forwarder. Some cleanup in servi…

### PR DESCRIPTION
**Pull request by tosan88**
…ce files

Tested in xp

### CODE REVIEW

File: splunk-forwarder.service [hunk 1]
```diff
@@ -6,17 +6,21 @@ TimeoutStartSec=1200
...
-ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=%p_)"-http > /dev/null 2>&1'
-ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p_)"-filter > /dev/null 2>&1'
-ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p_)"-http > /dev/null 2>&1'
-ExecStartPre=/usr/bin/docker pull coco/coco-splunk-http-forwarder
-ExecStartPre=/usr/bin/docker pull coco/coco-logfilter
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=%p-filter_)" > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=%p-http_)" > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p-filter_)" > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p-http_)" > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker pull coco/coco-splunk-http-forwarder'
```
Comment (BerniVarga) [inline splunk-forwarder.service]: Would it make sense to pull only if's not yet downloaded?

**Comment by BerniVarga** [comment]
+1

**Comment by catalin-ilea** [comment]
lgtm

File: splunk-forwarder.service (hunk 1 above)
Comment (tosan88) [inline splunk-forwarder.service]: We can't do it, as these are not versioned here & I didn't want to change that in this PR.

**Pull request merged**